logging/subscriber/src/helpers.ts

```diff
@@ -0,0 +1,5 @@
+import type { Level } from '@rango-dev/logging-types';
+
+export function isEnabled(maxLevel: Level, level: Level) {
+  return level >= maxLevel;
```

Comment:
NIT: I think the name **baseLevel** conveys the meaning better than **maxLevel**

Reply:
Agree. Renamed to `baseLevel`.